index.html

```diff
@@ -248,8 +249,20 @@ <h1>About <br />Myself</h1>
         </ul>
         <div class="dropdown"><li class="section_header">Frameworks</li><span>^</span></div>
         <div class="dropdown"> <li class="section_header">Skills</li><span>^</span>
-       
       </ul>
     </section>
+    <footer class="footer">
+        <form action="https://formspree.io/f/mjvzpbgj" method="post">
+            <h1 class="form_header">Contact me</h1>
+            <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                
+            <div class="form_inputs">
+                <input type="text" name="name" placeholder="Name" class="form_input" maxlength="30" required class="form_input">
+                <input type="email" name="email" placeholder="Email" class="form_input" required class="form_input">
+                <textarea name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here" class="form_input" required></textarea>
+            </div>
+          <button type="submit" class="form_button">Get in Touch</button>
```

Comment:
- [ ] Kindly add the button interaction in terms of hover and others just as on the Figma template
![image](https://user-images.githubusercontent.com/86200356/199261124-787e07d7-7bbc-463d-a87e-8146ef43daa3.png)